src/services/projects.js

```diff
@@ -452,7 +453,21 @@ module.exports = class ProjectsHelper {
 					}
 				}
 			}
-
+			//Add path in getDownloadUrl
+			if (result.certificate && result.certificate.base_template_url && result.certificate.base_template_id) {
```

Comment:
@praveenKDass while updating the project it should not store as a object in base_template_url, Could you please address that